main.js

```diff
@@ -12,6 +12,25 @@ import {
 
 let octokit;
 
+async function run() {
+  try {
+    const { token } = await getActionInputs();
+
+    octokit = getOctokit(token);
+
+    const pullRequest = await getPullRequest(context);
+    const fileDiffs = await diffAssets({ pullRequest });
+
+    await commentOnPR({ octokit, pullRequest, fileDiffs });
+  } catch (error) {
+    setFailed(error.message);
+  }
+}
+
+/** ************************************
```

Comment:
in all the "script"ish files I maintain, I find it _super_ beneficial to have the entry point front and center, and then have a visually distinguishable divider for all the supporting functions